distrib/initscripts/Makefile.am

```diff
@@ -105,7 +105,7 @@ install-data-hook:
 
 uninstall-startup:
 	-systemctl disable $(service_DATA)
-	rm -f $(addprefix $(DESTDIR)$(servicedir)/, $(service_DATA))
+	rm -f $(DESTDIR)$(servicedir)/$(service_DATA))
```

Comment:
Did you test that this syntax works for uninstalling all of the service files? Unlike netatalk3, $(service_DATA) is a list of space separated files.

Reply:
Well spotted, will revert the makefile change until I've found a fix...

Reply:
When I tinkered with this I went with the brute force approach:
```
	rm -f $(DESTDIR)$(servicedir)/a2boot.service \
	      $(DESTDIR)$(servicedir)/afpd.service \
	      $(DESTDIR)$(servicedir)/atalkd.service \
	      $(DESTDIR)$(servicedir)/cnid.service \
	      $(DESTDIR)$(servicedir)/papd.service \
	      $(DESTDIR)$(servicedir)/timelord.service
```

The repetition of code is not very elegant, but OTOH I don't think this list of services will change any time soon.

Reply:
Fixed in a (slightly) more elegant way...

Reply:
This works!

I'll run some tests on other platforms now and then merge if nothing broke.